build.gradle

```diff
@@ -58,7 +58,7 @@ dependencies {
     implementation 'net.wooga:unity-version-manager-jni:[1,2['
     implementation 'org.apache.commons:commons-compress:1.24.0'
 
-    testImplementation('org.jfrog.artifactory.client:artifactory-java-client-services:+') {
+    testImplementation('org.jfrog.artifactory.client:artifactory-java-client-services:2.19.0') {
```

Comment:
Do we need to pin the version?

Reply:
Unfortunately, yes. 2.19.1 uses groovy 4. Yeah breaking change on patch release.